### PR TITLE
Fix Swagger Gen and `make docs` command

### PR DESCRIPTION
**Pull request by mattverse**
Closes: #https://github.com/osmosis-labs/osmosis/issues/5445

## What is the purpose of the change
Fixes make docs command (which includes process for updating swagger) 

## Testing and Verifying
Tested in local environment

## Documentation and Release Note

  - [ ] Does this pull request introduce a new feature or user-facing behavior changes?
  - [ ] Changelog entry added to `Unreleased` section of `CHANGELOG.md`?

Where is the change documented? 
  - [ ] Specification (`x/{module}/README.md`)
  - [ ] Osmosis documentation site
  - [ ] Code comments?
  - [ ] N/A

### CODE REVIEW

File: scripts/generate-docs.sh [hunk 1]
```diff
@@ -5,46 +5,59 @@ set -eo pipefail
 mkdir -p ./tmp-swagger-gen
 
 # move the vendor folder to a temp dir so that go list works properly
-temp_dir="f29ea6aa861dc4b083e8e48f67cce"
+temp_dir=$(mktemp -d)
```
Comment (ValarDragon) [inline scripts/generate-docs.sh:8]: LOL did we hardcode a random looking directory, thats hilarious

**Review by ValarDragon — APPROVED** [review]
utACK, you planning to make a second PR updating swagger?

**Review by p0mvn — APPROVED** [review]
utACK

**Comment by p0mvn** [comment]
> utACK, you planning to make a second PR updating swagger?

I think it's here: https://github.com/osmosis-labs/osmosis/pull/5454

**Pull request merged**